File: app/static/javascript/noticeShare.js
const baseUrl = window.location.href.toString()
const facebook = document.getElementById('facebook')
const twitter = document.getElementById('twitter')
const whatsapp = document.getElementById('whatsapp')

facebook.setAttribute('href', `https://www.facebook.com/sharer/sharer.php?u=${baseUrl}`)
twitter.setAttribute('href', `https://twitter.com/intent/tweet?url=${baseUrl}`)
whatsapp.setAttribute('href', `https://api.whatsapp.com/send?text=${baseUrl}`)

File: app/utils/notice_share.py
from flask import request

def notice_share(social_media):
    url = request.url
    if social_media.lower() == 'facebook':
        return ('https://www.facebook.com/sharer/sharer.php?u=' + url)
    elif social_media.lower() == 'twitter':
        return ('https://twitter.com/intent/tweet?url=' + url)
    elif social_media.lower() == 'whatsapp':
        return ('https://api.whatsapp.com/send?text=' + url)
